### PR TITLE
validator tests, close #239

### LINKED ISSUE #239: Validator unit test coverage
validator coverage is 94%.
It should be raised to full coverage

now behaviors on low borders are uncovered:

validate_book empty rules have no coverage
Required, Lenght, Isbn on false return value have no coverage

### PR DESCRIPTION
**Pull request by webdewduck**

### CODE REVIEW

File: test_validator.py [hunk 1]
```diff
@@ -4,7 +4,17 @@
 
 
 class TestValidator(unittest.TestCase):
-    def setUp(self):
+
+    def test_validate_empty_rules(self):
+        self.validator = Validator()
+        self.assertEqual(True, self.validator.validate({}))
```
Comment (ArtemKrayevskiy) [inline test_validator.py:10]: Why here the validation must return True and on  test_validate_data_empty it returns False

File: test_validator.py [hunk 2]
```diff
@@ -93,6 +102,35 @@ def test_validate_phone_isbn(self):
...
+        validator = Validator()
+        validator.add({"code": [Isbn()]})
+        validator.add({"phone": [Length(min=3, max=13)]})
+        data = {
+            "code": "111",
+            "phone": "+"
+        }
+        result = validator.validate(data)
+        self.assertEqual(False, result)
+        self.assertEqual('Wrong phone', validator.errors.get('phone').pop(0))
```
Comment (ArtemKrayevskiy) [inline test_validator.py]: Why here the Error which costumer got is 'Wrong phone' if phone rule wasn't mentioned in rules. Isn't it just Wrong length?

File: test_validator.py (hunk 1 above)
Comment (webdewduck) [reply test_validator.py:10]: I am trying to check an empty validator.
When the validator has no rules to validate, any data (empty or not empty) should be valid.
The correct behavior example is on line 12

File: test_validator.py (hunk 2 above)
Comment (webdewduck) [reply test_validator.py]: You are absolutely correct, my bad

**Review by savag21 — APPROVED** [review]

**Pull request merged**